File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.c
int minOperations(char* s) {
    int n, c = 0, j = 0;
    for(n = 0; s[n] != '\0'; n++) {
        if(s[n] - '0' == j) c++;
        j ^= 1;
    }
    return c < n - c ? c : n - c;
}

File: 1884-minimum-changes-to-make-alternating-binary-string/minimum-changes-to-make-alternating-binary-string.js
var minOperations = function(s) {
    let n, c = 0, j = 0;
    for(n = 0; n < s.length; n++, j ^= 1) {
        if(s[n] - '0' == j) c++;
    }
    return c < n - c ? c : n - c;    
};